Create a new handbook page for curating the editor experience

annezazu opened the pull request:
## What?

This proposes a v1 of a new page that compiles various methods for curating the editor experience. Thanks to @justintadlock @carolinan @bph for chiming in/reviewing! In the future, I'd love to see a section with specific recommendations for Agencies and Enterprise level folks. My hope is that an initial version can get out into the open and folks in those spaces can contribute back with use studies, feedback on what would be helpful, etc. 

## Why?

Right now, the information around how best to curate the experience/tailor it to your needs is spread across multiple different resources. This seeks to pull them into one place and explain how they each can be used. 

## Please help with this PR!

I still struggle at times to get things in place for new docs so, if you're game to help, it would be lovely for folks to contribute to this PR both to move it forward and suggest changes.

annezazu committed:
Create a new handbook page for curating the editor experience

This proposes a v1 of a new page that compiles various methods for curating the editor experience.

annezazu committed:
Update manifest.json to include new page

annezazu committed:
Update toc.json for new handbook page

annezazu commented:
Noting that I definitely added the visuals incorrectly (didn't see the assets folder). I'll try to figure this out now :)

annezazu committed:
Add screenshots to assets folder

annezazu committed:
Add example video to assets

Had to speed this video up in order for it to be small enough to include.

annezazu committed:
Updating asset references

I can't seem to figure out why the video won't properly render. When I look at the individual file, it seems the size is still too big to display. If I don't add it to the assets file and just drop it in, it works though.

annezazu commented:
I can't seem to figure out why the video won't properly render. When I look at the individual file, it seems the size is still too big to display. If I don't add it to the assets file and just drop it in, it works though. If anyone can help here, that would be rad :)

fabiankaegy commented on docs/how-to-guides/curating-the-editor-experience.md:
```diff
@@ -0,0 +1,314 @@
+
+Beyond defining default values, using theme.json allows you to also remove options entirely and instead rely on what the theme has set in place. Below is a visual showing two extremes with the same paragraph block: 
+
+![Image of restricted interface](https://github.com/WordPress/gutenberg/blob/annezazu-curate-new-doc/docs/assets/Locking%20comparison%20visual.png)
```
I think the URL of the image already needs to point to the path of the image once it has been merged to trunk.

annezazu replied:
Ah! Okay. Let me update that :) I just was worried they wouldn't appear if I reference them in trunk without them being there first. Makes sense though.

annezazu replied:
Updated but, as you can see now, the images don't show up. I guess that's what has to be done but I'm not sure what folks have done previously.

fabiankaegy reviewed (commented):
I really like this guide. It is a great overview of the various options that one has :)

ryanwelcher committed:
Rebuilds docs manifest.

annezazu committed:
Updating assets to refer to path to trunk rather than my PR

annezazu committed:
Resolve conflict with manifest.json

Not sure if this is the right way to resolve a conflict with trunk but this is my attempt :)

annezazu committed:
Minor spacing update to manifest.json

gziolo commented:
You need to run `npm run docs:build` to regenerate the manifest file used at https://developer.wordpress.org/block-editor/.

ryanwelcher committed:
Rebuild the docs.

ryanwelcher commented:
> You need to run `npm run docs:build` to regenerate the manifest file used at https://developer.wordpress.org/block-editor/.

@annezazu I took the liberty of rebuilding the docs and pushing the changes up.

ryanwelcher committed:
Merge branch 'trunk' into annezazu-curate-new-doc

ryanwelcher reviewed (approved):
LGTM! 👍
